composites/basic/Notification.js

```diff
@@ -0,0 +1,111 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+import { injectIntl, intlShape, defineMessages } from "react-intl";
+
+import Paper from "./Paper";
+import colors from "../../style-guide/colors.json";
+import { Icon } from "../../composites/Plugin/Shared/components/Icon";
+import { times } from "../../style-guide/svg";
+
+const messages = defineMessages( {
+	buttonAriaLabel: {
+		id: "notification.buttonAriaLabel",
+		defaultMessage: "Dismiss this notice",
+	},
+} );
+
+const NotificationContainer = styled.div`
+	display: flex;
+	align-items: center;
+	padding: 20px;
+
+	h1, h2, h3, h4, h5, h6 {
+		font-size: 1.4em;
+		line-height: 1;
+		margin: 0 0 4px 0;
+	}
+
+	p:last-child {
+		margin: 0;
+	}
+`;
+
+const NotificationImage = styled.img`
+	flex: 0 0 ${ props => props.imageWidth ? props.imageWidth : "auto" };
+	height: ${ props => props.imageHeight ? props.imageHeight : "auto" };
+	margin-right: 18px;
```

Comment:
to fit the 8px grid

Reply:
Same as above: in the provided mockup, the distance is 20px. I've use 18px to compensate the font metrics. If you measure the distance from the rendered image and the rendered title, it's 20px.
Are we sure we want to change this? The meta boxes below use a 20px padding.

Reply:
To clarify: font metrics and font rendering engines matter. If we use a distance (whether it's padding or margin) between the image and the text, this applies to the text _container_. The actual text is rendered with a few pixel distance from its container, and that's normal on the web, it just works this way. It also depends on the actual font face used and even from the shape of the first letter:

<img width="642" alt="screen shot 2017-10-16 at 13 37 35" src="https://user-images.githubusercontent.com/1682452/31610294-c219b93e-b277-11e7-8478-c30f76de0a67.png">

Setting the exact same value on the left and on the right, will actually result in an uneven perceived spacing.

Reply:
That's interesting, so 18px is actually 20px visually? I hadn't heard about that before. Is this compensated consistently throughout our existing UI? If so, okay, otherwise, I'd prefer coding consistency over a small visual inconsistency.